units/initrd-battery-check.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Check battery level during early boot
+DefaultDependencies=no
+After=initrd-root-fs.target
```

Comment:
this uses udev, so it needs to be after systemd-udevd.service

Reply:
> this uses udev, so it needs to be after systemd-udevd.service

Thank you sir.